### PR TITLE
Give panels a face lift

### PR DESCRIPTION
**Pull request by jackbentley**
Reduce panel title size
Add a separator bar between title and content
Increase panel default padding

**Before**
Screen Shot 2019-12-04 at 14 33 01

**After**
Screen Shot 2019-12-04 at 14 23 40

### CODE REVIEW

File: stylesheets/_component.panels.scss [hunk 1]
```diff
@@ -7,18 +7,25 @@
     &__title {
         border: 0;
         clear: none;
-        font-size: $font-size-xlarge;
-        line-height: 32px;
+        font-size: $font-size-large * 1.1;
+        line-height: $font-size-large * 1.3;
```
Comment (jamesjacobs) [inline stylesheets/_component.panels.scss]: Line height needs to be 1.5 times the font-size to meet WCAG 1.4.12 Text Spacing (AA). May also need to tweak the heading icon position to ensure vertical alignment after increasing line-height.

File: stylesheets/_component.panels.scss [hunk 2]
```diff
@@ -7,18 +7,25 @@
...
         margin-bottom: 0;
-        padding: 8px 5px 0 10px;
+        padding: 12px $padding-large-horizontal;
+        background: rgba(0, 0, 0, .01);
```
Comment (jamesjacobs) [inline stylesheets/_component.panels.scss]: Properties within rules should be alpha ordered.

File: stylesheets/_component.panels.scss [hunk 3]
```diff
@@ -108,8 +115,8 @@
 }
 
 .panel--padded {
-    padding-top: 24px;
-    padding-bottom: 24px;
+    padding-top: ($padding-large-vertical * 2);
```
Comment (jamesjacobs) [inline stylesheets/_component.panels.scss]: Properties within rules should be alpha ordered.

File: stylesheets/_component.panels.scss (hunk 1 above)
Comment (Stanton) [reply stylesheets/_component.panels.scss]: Doesn't need to be, just has to not break if the line height is set to 1.5.

File: stylesheets/_component.panels.scss (hunk 3 above)
Comment (jackbentley) [reply stylesheets/_component.panels.scss]: Should probably add an automated test for this 😄

File: stylesheets/_component.panels.scss (hunk 2 above)
Comment (jamesjacobs) [reply stylesheets/_component.panels.scss]: Fixed

File: stylesheets/_component.panels.scss (hunk 3 above)
Comment (jamesjacobs) [reply stylesheets/_component.panels.scss]: Fixed

**Pull request merged**